<ui version="4.0" >
 <author>Aaron Racicot</author>
 <class>AllVariablesEditGui</class>
 <widget class="QDialog" name="AllVariablesEditGui" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Variable Library</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QWidget" native="1" name="variableBox" >
     <layout class="QGridLayout" name="gridLayout" />
    </widget>
   </item>
   <item>
    <widget class="QWidget" native="1" name="widget" >
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>45</height>
      </size>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>441</width>
          <height>24</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="addNewVariable" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string>Add New Variable</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="checkAllVariables" >
        <property name="toolTip" >
         <string>Checks all variable syntax and makes sure that it can be computed on the baseyear data.</string>
        </property>
        <property name="text" >
         <string>Validate All Variables</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelWindow" >
        <property name="text" >
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
